common/app/routes/challenges/utils.js

```diff
@@ -90,7 +90,7 @@ function logReplacer(value) {
     const replaced = value.map(logReplacer);
     return '[' + replaced.join(', ') + ']';
   }
-  if (typeof value === 'string' && !value.startsWith('//')) {
+  if (typeof value === 'string' && !RegExp('^//').test(value)) {
```

Comment:
So, to use the constructor method of creating a `RegExp` we need the `new` keyword. This ensures it is has direct inheritance from `RegExp.prototype` and we are not leaving ourselves open to bugs or attacks. Also, it is better to escape the `/`'s for the same reasons.

You can just create a literal RegExp and test with 
```js
/^\/\//.test(value)

// -------or-------

const inlineCommentRE = /^\/\//;
inlineCommentRE.test(value)
```
which will sort of the inheritance for us. Note that the `/`'s are still escaped. 

Thanks for all your effort in this PR, I think one last change and it should be ready to merge ✨